portal-api/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dtfs-portal-api",
```

Comment:
This is the only one with `dtfs` in the name as a suffix
```suggestion
  "name": "portal-api",
```

Reply:
Have updated.